src/PerformanceTests/Persistence.V5.InMemory.V5/Persistence.V5.InMemory.csproj

```diff
@@ -7,8 +7,8 @@
     <ProjectGuid>{D8CB2DA9-4AE4-417C-89C3-00555F5E1F47}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Persistence.V5.InMemory.V5</RootNamespace>
-    <AssemblyName>Persistence.V5.InMemory.V5</AssemblyName>
+    <RootNamespace>Persistence.V5.InMemory</RootNamespace>
```

Comment:
@ramonsmits I restored the versions (with a small change, replaced dot with underscore) for the downstream dependencies. For the built-in ones (like InMemory and MSMQ) I omitted the version. If we ever put them outside of core, they will probably be versioned at 1.0.0, so until then calling it v6 is confusing.